### ISSUE TITLE
Markup error: `diff` code block

### ISSUE THREAD

**Issue by liuyinz** [jrblevin/markdown-mode#631]
## Expected Behavior
```markdown
```diff
- plugins=(...)
+ plugins=(... fzf-collection)
```

When enable hide markup, the code block shouldn't be changed:
```diff
- plugins=(...)
+ plugins=(... fzf-collection)
```

## Actual Behavior
The "+" and "-" in 'diff' code block hide too:

- plugins=(...)
+ plugins=(... fzf-collection)

## Steps to Reproduce

1. `emacs -Q`
2. `(markdown-mode 1)`
3. `(markdown-toggle-markup-hiding)`

## Software Versions

- Markdown Mode: "markdown-mode, version 2.5-dev"
- Emacs: GNU Emacs 28.0.50 (build 1, x86_64-apple-darwin20.6.0, NS appkit-2022.60 Version 11.5 
- OS: macOS Big Sur

**Comment by syohex** [comment]
@liuyinz I have merged #632. Could you check latest version ?

**Comment by liuyinz** [comment]
Solved, Thanks!
